README.md

```diff
@@ -5,6 +5,10 @@ You can find the source for [v4 on the corresponding branch](https://github.com/
 
 🚧 **WARNING: WORK IN PROGRESS** 🚧
 
+> ⚠️ **WARNING** ⚠️
```

Comment:
You need to add empty lines in order for the warnings to be on their own line, which I think was your intention

Reply:
Thanks for pointing that out. I added the empty line after the first "Warning" and removed the closing one altogether to follow the same format used in another one below.

![screenshot-github com-2023 02 13-05_56_25](https://user-images.githubusercontent.com/52650/218374026-5b2a52ee-d8a5-44dd-aa28-ec031b22b87b.png)